Jellyfin.Api/Controllers/ItemUpdateController.cs

```diff
@@ -158,7 +158,10 @@ public ActionResult<MetadataEditorInfo> GetMetadataEditorInfo([FromRoute, Requir
             ParentalRatingOptions = _localizationManager.GetParentalRatings().ToList(),
             ExternalIdInfos = _providerManager.GetExternalIdInfos(item).ToArray(),
             Countries = _localizationManager.GetCountries().ToArray(),
-            Cultures = _localizationManager.GetCultures().ToArray()
+            Cultures = _localizationManager.GetCultures()
```

Comment:
Since both usages of `GetCultures()` filter by distinct and then order I think `_cultures` should just store this list